src/components/Autocomplete/AutocompleteSearch.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+    <div>
+        <div class="dropdown">
+            <input
+                type="text"
+                class="form-control w-full px-5 custom-select bg-white rounded-md py-4 font-semibold text-lg"
+                v-model="inputValue"
+                @input="searchEvent"
+            />
+            <div v-show="inputValue && apiLoaded" class="dropdown-list">
```

Comment:
inputValue and apiLoaded is quite confusing at first glance without going through the whole flow.

I suggest something more descriptive.

inputValue => productName as it is in the query param.
apiLoaded => productFetched  or productRetrieved etc

searchEvent can also be renamed to SearchProductByName or searchProduct or searchProductBySomething.....

Reply:
noted